### PR TITLE
feat(icons): added `droplet-off` icon

### PR DESCRIPTION
**Pull request by jguddas**
## What is the purpose of this pull request?
- [x] New Icon

### Description
Added new `droplet-off` icon.

### Icon use case

See #2632

### Concept

- [x] I have provided valid use cases for each icon.
- [x] I have not added any a brand or logo icon.
- [x] I have not used any hate symbols.
- [x] I have not included any religious or political imagery.

### Author, credits & license

- [ ] The icons are solely my own creation.
- [x] The icons were originally created in #<issueNumber> by @colebemis
- [x] I've based them on the following Lucide icons: `droplet`
- [ ] I've based them on the following design: 

### Naming

- [x] I've read and followed the [naming conventions](https://lucide.dev/guide/design/icon-design-guide#naming-conventions)
- [x] I've named icons by what they are rather than their use case.
- [x] I've provided meta JSON files in `icons/[iconName].json`.

### Design

- [ ] I've read and followed the [icon design guidelines](https://lucide.dev/guide/design/icon-design-guide)
- [ ] I've made sure that the icons look sharp on low DPI displays.
- [ ] I've made sure that the icons look consistent with the icon set in size, optical volume and density.
- [ ] I've made sure that the icons are visually centered.
- [ ] I've correctly optimized all icons to three points of precision.

## Before Submitting
- [x] I've read the [Contribution Guidelines](https://github.com/lucide-icons/lucide/blob/main/CONTRIBUTING.md).
- [x] I've checked if there was an existing PR that solves the same issue. **And there is an existing PR with the same issue at #2632**

### CODE REVIEW

**Comment by karsa-mistmere** [comment]
How does this relate to #2632 ?

**Comment by jguddas** [comment]
> #2632

It's the automatically generated version @Footagesus painstakingly handcrafted.
The upside of having this automated version is that there are no accidental errors introduced, rounding errors, missing contributors, etc.

**Comment by Footagesus** [comment]
Okay, I don't mind. The important thing is that the icon was added.

File: icons/droplet-off.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "colebemis",
+    "ericfennis",
+    "csandman",
+    "johnletey",
+    "jguddas"
```
Comment (ericfennis) [inline icons/droplet-off.json:8]: Shouldn't we add @Footagesus here as well to give him some credits as well?

Comment (jguddas) [inline icons/droplet-off.json:8]:
```suggestion
    "jguddas",
    "Footagesus"
```

**Comment by Footagesus** [comment]
Yo ty

**Pull request merged**